src/db/MariadbIfc.py

```diff
@@ -427,6 +426,34 @@ def getProfiles(self,
 
         return results
 
+    def getProfiles(self,
+                    user_id : int,
+                    name : str) -> list:
+        """Returns profiles matching a given name filter for a given user.
+
+            Input: self - Pointer to the current object instance.
+                   user_id - user ID to interrogate for profiles.
+
+            Output: list - A list of all profiles found, if any.  An empty list if not.
+        """
+        cmd     = ""
+        cursor  = self.con.cursor(buffered=False)
+        results = []
+
+        self.db_log.info(f"Getting profiles for user {user_id}")
```

Comment:
The log message should be unique so its easy to tell which function was called from the log alone.  Maybe something like "Searching for profiles matching {name} for {user_id}"?